### ISSUE TITLE
Ready translation file for Biomes o'Plenty module on Polish pl_pl.lang

### ISSUE THREAD

**Issue by EliogabalusPL** [Glitchfiend/BiomesOPlenty#1117]
--------------------------------------------------------------------------------

**BEFORE SUBMITTING!**

You must ensure the following before reporting an issue with the mod.
For feature requests, you may delete this template and create your own.

--------------------------------------------------------------------------------

- [ ] Biomes O' Plenty is updated to the latest available version.
- [ ] This issue occurs with Minecraft 1.12. 

**We *DO NOT* support Minecraft versions 1.7.10, 1.8.9, 1.9.4, 1.10.2, 1.11 or anything below that!**  

**YOUR ISSUE WILL AUTOMATICALLY BE CLOSED IF IT IS PERTAINING TO THOSE VERSIONS!**

--------------------------------------------------------------------------------

- [ ] Forge is updated to the latest available version.

- [ ] This issue occurs *WITHOUT* Optifine, Sponge or similar mods installed.

**Mods such as Optifine and Sponge interfere with the normal functioning of the game by modifying its code.**

**We do not support these mods in any way as they can introduce bugs which would not normally be present with**
**only Forge installed.**

--------------------------------------------------------------------------------

- [ ] This issue can be reproduced with ONLY Biomes O' Plenty and any other affected mods.

- [ ] All other mods which appear to be involved are updated to their latest versions.

- [ ] I have included any relevant logs and screenshots below.

**If this is a compatibility issue with another mod, you *MUST* test with *ONLY* Biomes O' Plenty and that specific mod.**

**Logs containing masses of mods are extremely unhelpful, and it is unlikely that your issue will be addressed.**

--------------------------------------------------------------------------------

- [x] This issue does not occur in a modpack. 

**Issues with modpacks should be reported to the modpack author(s). As noted above, mod compatibility issues must**
**occur with the latest versions of both Biomes O' Plenty and the affected mod.** 

**The issue *MUST* be tested with *ONLY* Biomes O' Plenty and the affected mod.**

--------------------------------------------------------------------------------

*Once you have read and verified the above, you may delete all text up to this point.*

#### Issue Description:
2 months ago I was interested in your very interesting module. Unfortunately, it turned out that it lacked a translation file in Polish, so I decided to translate en_us.lang file into Polish.
If you would like to add this translation to your work, maybe a bunch of Biomes O'Plenty devotees from my country will be grateful to you.
I am sending a link to the translation file here - https://1drv.ms/u/s!At_Q76MtyFIHhxcVRg_Ce5OV0HRV (this is the final version of the translation)

It is already available on several portals for the Minecraft modules, but it would be much more convenient if it was directly in the module than was added manually.
https://craftportal.pl/forum/index.php?/topic/103213-t%C5%82umaczenia-popularnych-mod%C3%B3w-na-pl/?p=1181951
2. http://forum.minecraft.pl/topic/30223-t%C5%82umaczenia-popularnych-mod%C3%B3w-na-pl/
As you can see in the links to the forums about Minecraft modules I am translating a lot of modules into Polish, so it is not new to me.

#### Steps to reproduce:

1. After verifying that I did not use any illegal words / expressions (eg through Google Translator) add the pl_pl.lang file to the module.
2. 
3. 

#### Additional Information:

- fml-client-latest.log or fml-server-latest.log:
- Screenshots:

____
#### Affected Versions (Exact numbers, do *NOT* use "latest"):

- Biomes O' Plenty: Translation file checked in "BiomesPlenty-1.12-7.0.0.2291-universal" (or works in other versions I did not check)
- Minecraft: 1.12.1
- Forge: 1.12.1

**Comment by Forstride** [comment]
Please submit this as a pull request: https://github.com/Glitchfiend/BiomesOPlenty/pulls

**Comment by EliogabalusPL** [comment]
The trouble is that I do not know how to add to the file Github pl_pl.lang, for this I sent to this file link.

**Comment by Forstride** [comment]
google it.  it's really not that hard

**Comment by EliogabalusPL** [comment]
"It's really not hard" - unfortunately for me it is a bit difficult. Maybe if I were 30 years old and "I would suck my knowledge out of breast milk" ...

I do not know if I've added the contents of the file pl_pl.lang to Gihub https://github.com/Glitchfiend/BiomesOPlenty/compare/BOP-1.12.x-7.0.x...EliogabalusPL:patch-1.
I made a request at the request of my daughter, who is a fan of Minecraft and your module. Unfortunately, I never had to deal with Gitub, and rather counted on your help instead of sending me to Google, because even the phrase "add a fork" is incomprehensible to me, and I did not want to mess up Github Biomes O'Plenty.
